src/components/ContentUploader/OverallUploadsProgressBar.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @flow
+ * @file Overall uploads progress bar
+ */
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from '../messages';
+import ProgressBar from './ProgressBar';
+import { VIEW_UPLOAD_IN_PROGRESS, VIEW_UPLOAD_SUCCESS, VIEW_ERROR, VIEW_UPLOAD_EMPTY } from '../../constants';
+import type { View } from '../../flowTypes';
+
+import './OverallUploadsProgressBar.scss';
+
+type Props = {
+    isVisible: boolean,
+    percent: number,
+    onClick: Function,
+    onKeyDown: Function,
+    view: View
+};
+
+/**
+ * Get upload status
+ *
+ * @param {View} view
+ * @return {FormattedMessage|string}
+ */
+const getUploadStatus = (view) => {
+    switch (view) {
+        case VIEW_UPLOAD_IN_PROGRESS:
+            return <FormattedMessage {...messages.uploadsManagerUploadInProgress} />;
+        case VIEW_UPLOAD_SUCCESS:
+        case VIEW_UPLOAD_EMPTY:
+            return <FormattedMessage {...messages.uploadsManagerUploadComplete} />;
+        case VIEW_ERROR:
+            return <FormattedMessage {...messages.uploadsManagerUploadFailed} />;
+        default:
+            return '';
+    }
+};
+
+const getPercent = (view, percent) => {
+    if (view === VIEW_UPLOAD_SUCCESS || view === VIEW_UPLOAD_EMPTY) {
+        return 100;
+    } else if (view === VIEW_ERROR) {
+        return 0;
+    }
+
+    return percent;
+};
+
+const OverallUploadsProgressBar = ({ percent, view, onClick, onKeyDown, isVisible }: Props) => (
+    <div
```

Comment:
Why not use a PlainButton here?
So that we aren't adding onClicks to divs.

Reply:
done

Reply:
actually, still keeping it as a div, because I don't want the styles that come with PlainButton, and I don't want the browser default styling for button